mimic/canned_responses/noit.py

```diff
@@ -0,0 +1,123 @@
+"""
+Canned response for Noit
+"""
+import time
+from mimic.canned_responses.noit_metrics_fixture import (metrics_common_template,
+                                                         metrics)
+
+
+noit_cache = {}
```

Comment:
nope nope nope

We don't have a session to store per-user information on.  That's... weird, and I don't know how noit manages to do what it does without authentication. However, there's a relatively simple process for de-`_cache`-ifying this and making it so we don't depend on global state everywhere:

  1. take every function in this module that touches `noit_cache` and put them as methods on an object
  2. instantiate that object from within `NoitApi` and store it as an object on `self`

We can figure out how noit does authentication and sessions later, but when we do, we can then just take that object and stick it in a session, rather than having to change all the code that touches this global.